WordPress/src/main/res/layout/login_site_address_screen.xml

```diff
@@ -1,88 +1,32 @@
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:layout_marginBottom="@dimen/margin_extra_large"
+        tools:text="@string/enter_site_address" />
 
-            <android.support.design.widget.TextInputLayout
-                app:theme="@style/LoginTheme.EditText"
-                android:id="@+id/login_site_address_layout"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_below="@id/label"
-                android:layout_toRightOf="@+id/login_icon_globe"
-                android:layout_marginLeft="@dimen/margin_extra_large">
-
-                <android.support.design.widget.TextInputEditText
-                    android:id="@+id/login_site_address"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:hint="@string/login_site_address"
-                    android:inputType="textUri"
-                    android:drawablePadding="@dimen/margin_large"/>
-            </android.support.design.widget.TextInputLayout>
-
-            <RelativeLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_alignParentBottom="true">
-
-                <Button
-                    android:theme="@style/LoginTheme.Button"
-                    style="@style/Widget.AppCompat.Button.Colored"
-                    android:id="@+id/login_next_button"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_centerVertical="true"
-                    android:layout_alignParentRight="true"
-                    android:enabled="false"
-                    android:text="@string/next" />
-
-                <TextView
-                    android:id="@+id/login_site_address_help"
-                    style="@style/Base.TextAppearance.AppCompat.Caption"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_toLeftOf="@+id/login_next_button"
-                    android:layout_marginRight="@dimen/margin_extra_extra_extra_large"
-                    android:layout_centerVertical="true"
-                    android:layout_alignParentLeft="true"
-                    android:textColor="@color/blue_wordpress"
-                    android:text="@string/login_site_address_help"/>
-            </RelativeLayout>
-        </RelativeLayout>
-    </ScrollView>
-</RelativeLayout>
+    <org.wordpress.android.widgets.WPLoginInputRow
+        android:id="@+id/login_site_address_row"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/login_site_address"
+        android:inputType="textUri"
+        android:imeOptions="actionNext"
```

Comment:
Should the `flagNoExtractUi` value be added to the `android:imeOptions` attribute to disable the fullscreen extract interface when tapping the input view in landscape orientation?  See the screenshot below for illustration.

![extract_on](https://user-images.githubusercontent.com/3827611/28400383-fbb3cdc6-6ccf-11e7-9b06-1d5a00dac9a2.png)

The layout would need some extra work since using `android:imeOptions="actionNext|flagNoExtractUi"` would produce some odd scrolling behavior as shown in the screenshot below.

![extract_off](https://user-images.githubusercontent.com/3827611/28400419-4009da42-6cd0-11e7-9933-1307c423f484.png)